### DISCUSSION TITLE
IPC not detecting ??

### DISCUSSION THREAD

**Discussion by MrTerrorM** [ppy/osu#15749] [Q&A] [answered]
Hello
Why it doesn't work?

![image](https://user-images.githubusercontent.com/91147220/142935502-583b737b-53aa-4931-bf6a-6f49f2124575.png)
![image](https://user-images.githubusercontent.com/91147220/142935516-9bd82a08-4a71-438c-922d-9e3b7c36f117.png)
![image](https://user-images.githubusercontent.com/91147220/142935539-8d8f05da-8608-4607-a3be-46dd51b9c5de.png)

**Comment by bdach** [comment, marked as answer]
your screenshot above shows extensions on unhidden files but not on hidden ones. you may have accidentally named the ipc file `ipc.txt.txt` because of this.

please try removing the `.txt` at the end and trying agian.

**Reply by MrTerrorM** [reply]
I downloaded new osu!.exe and it also worked.
Thanks for helping :D
